invenio_pages/services/service.py

```diff
@@ -7,29 +7,123 @@
 
 """Page Service API."""
 
-from flask import abort
-from invenio_access.permissions import system_identity
 from invenio_records_resources.services import RecordService
-from sqlalchemy.orm.exc import NoResultFound
+from invenio_records_resources.services.base import LinksTemplate
+from invenio_records_resources.services.base.utils import map_search_params
+from invenio_records_resources.services.uow import unit_of_work
+from sqlalchemy import func
 
-from invenio_pages.records.models import PageModel as Page
+from ..records.models import PageModel as Page
 
 
 class PageService(RecordService):
     """Page Service."""
 
-    def read_url(self, url):
-        """Retrieve a page."""
-        return Page.get_by_url(url)
+    @unit_of_work()
+    def create(self, identity, data, uow=None):
+        """Create a page."""
+        self.require_permission(identity, "create")
```

Comment:
do we have a test case to make sure only superuser(and not administrator) can create and delete?